resume/resume.org

```diff
@@ -20,11 +20,12 @@ Not knowing how something works has never stopped me from achieving anything.
 * Skills / Experience Summary
 
 - Entirely self taught with a background in Computer Science
-- Fast learner with the ability to teach myself and others new skills quickly
-- Sixteen years experience in the IT industry in a variety of businesses
-- Highly skilled in automation tools like Kubernetes, Puppet, and Docker
-- Skilled in CI/CD tools and methodologies
-- Experience with Agile methodologies and practices
+- Fast learner with the ability to teach myself new skills quickly
```

Comment:
Does this come across as redundant to you? "Fast learner" and  "ability to teach myself new skills quickly" sound like they're trying to say the same thing.

Reply:
Good point. Yep it does. I wIll fix it.